dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/AbstractPagingAndSortingRepositoryImpl.java

```diff
@@ -167,6 +181,18 @@ protected String getWhereClause(FilterCriterion<?> fc)
               .append(convertToSqlString(fc.getValue()))
               .append(")");
           break;
+        case GREATER_THAN_OR_NOT_SET:
```

Comment:
GitHub läßt mich die betroffene Stelle leider nicht direkt kommentieren, aber den Code in `case NOT_IN` könnte man ebenfalls vereinfachen:

```
query.append(
              fc.getValues().stream().map(this::convertToSqlString).collect(Collectors.joining(",")));
```